### DISCUSSION TITLE
v0.9.1

### DISCUSSION THREAD

**Discussion by 0xCUB3** [2009-Nissan-Cube/About-This-Hack#40] [Announcements]
Just some quick tweaks and bug fixes!

## What's Changed
- Fixed some formatting issues with dual-GPUs
- The "Bootloader" category now shows "Apple iBoot" for Apple Silicon Macs
- Resolved a bunch of pesky errors

*Please note that About This Hack is still in beta, so bugs exist. If you do find anything that isn't covered in the README, please [open an issue](https://github.com/0xCUB3/About-This-Hack/issues).*

**Full Changelog**: https://github.com/0xCUB3/About-This-Hack/compare/0.9.0...0.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/0xCUB3/About-This-Hack/releases/tag/0.9.1'>v0.9.1</a>.</em>

**Comment by kaoskinkae64** [comment]
missing compiled version it is important to keep that on (.......) since those of us who use the beta versions of the different systems available, both Ventura and Sonoma, it is important to know which version we are using
